odl/test/trafos/fourier_test.py

```diff
@@ -631,6 +631,53 @@ def char_rect(x):
             assert all_almost_equal(ft.adjoint(ft(char_rect)), discr_rect)
 
 
+def _test_adjoint(operator):
+    """Test adjoint of operator satisfies the definition."""
+    x = noise_element(operator.domain)
+    y = noise_element(operator.range)
```

Comment:
> I seem to be getting a factor 2 across the board, which seems to be the same as we saw in the bug report #732

That could come from the roughly halved range in the real-to-complex case. Maybe that case needs a scaling? It's not exactly 2, it's more like `n / (n // 2 + 1)` probably.